src/routes/EducationOpportunity/loader.ts

```diff
@@ -48,91 +48,91 @@ const loader = (async ({ request, params }) => {
               tyhjia: 0,
               arvot: [
                 {
-                  arvo: 'http://data.europa.eu/esco/skill/023faef3-0633-436d-8b91-389c45c1421c',
+                  arvo: 'https://data.europa.eu/esco/skill/023faef3-0633-436d-8b91-389c45c1421c',
```

Comment:
Now that we get koulutusmahdollisuudet from the backend, maybe this mock data can be removed completely.

Reply:
Maybe this can be done in separate PR or other that is relevant to that actual change.